sites/partners/src/listings/PaperListingForm/index.tsx

```diff
@@ -29,6 +29,7 @@ type FormListing = ListingCreate & ListingUpdate
 type ListingFormProps = {
   listing?: FormListing
   editMode?: boolean
+  isAdmin?: boolean
```

Comment:
very nit but:
Another possible option could be to name and the prop in `FormListingData` this something like `hasFullEditRights` in case the concepts of "admin" and "HRD user that can edit stuff" diverge in the future. That also, to me at least, adds clarity to the places it's used down the tree as to _why_ a field might be disabled.

Alternatively, maybe there's a way to make `editMode` an enum that holds both the concepts of "editing vs new listing" and "regular vs full rights user"

Reply:
In order to support #83, I could also see this being an enum called something like "editAccessLevel" that contains things like `allFields`, `availabilityOnly`, `propertyManager`